single_tech_samples/datafactory/sample2_data_pre_processing_with_azure_batch/src/sample-processor/README.md

```diff
@@ -0,0 +1,75 @@
+# Sample Processor
+
+This is a sample image which is buid from ros:noetic base image. It extracts the contents of a given bag file to an output path dirctory. Both the input file path and the output path are provided as parameters.
+
+## Steps To build a docker image
+
+1. Go to the folder `src/sample-processor`
+
+    ```shell
+    cd ./single_tech_samples/datafactory/sample2_data_pre_processing_with_azure_batch/src/sample-processor
+    ```
+
+2. Build a docker image by running
+
+    ```shell
+    docker build . -t sample-processor:latest
+    ```
+
+### Testing your image locally
+
+Run the following command, which mounts src/sample-processor/data to your container.
+
+`Make sure to update <YOURPATH> with your clonned directory path`
+
+```shell
+docker run --rm --mount type=bind,source=<YOURPATH>/src/sample-processor/data,target=/data sample-processor:latest bash -c "source /opt/ros/noetic/setup.bash&&python3 /code/app.py /data/raw/sample.bag /data/extracted && rosbag info '/data/raw/sample-data.bag' > /data/extracted/rosbagInfo.txt"
```

Comment:
This didn't work for me.

- There is no "/code" folder for /code/app.py".
- Even after fixing it, somehow it didn't work.

Reply:
Yes, it was missing the parameter names -i and -o